src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public String getColor() {
+        return new ColorSupplier().getRandomColor();
+    }
+
+    public int getNumber() {
+        return new Number().randomNumberForBall();
+    }

```

Comment:
a class must not have getters
create a constructor or setters to set the fields
```suggestion
```

Reply:
how is it, during the tests it was indicated to me that the number and color should be private, and if you remove the getters, how to get the information?